addons/sale_stock/models/sale_order_line.py

```diff
@@ -43,7 +43,8 @@ def _compute_qty_to_deliver(self):
 
     @api.depends(
         'product_id', 'customer_lead', 'product_uom_qty', 'product_uom', 'order_id.commitment_date',
-        'move_ids', 'move_ids.forecast_expected_date', 'move_ids.forecast_availability')
+        'move_ids', 'move_ids.forecast_expected_date', 'move_ids.forecast_availability',
+        'warehouse_id')
```

Comment:
I would remove `rental` in the commit message as it is also useful in sale_stock in general